.github/workflows/black.yml

```diff
@@ -1,11 +1,36 @@
 name: Black Formatter
-on: [push, pull_request]
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches_ignore: []
+
 jobs:
-  black:
+  formatting:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - name: Black Code Formatter
-        uses: lgeiger/black-action@master
+      - uses: actions/checkout@v2
+
+      - name: Setup black linter
+        run: conda create --quiet --name black black
+
+      - name: Lint python code
+        run: |
+          export PATH="/usr/share/miniconda/bin:$PATH"
+          source activate black
+          black --check buildtest tests/*.py
+
+      - name: Derive organization name
+        if: github.event_name == 'pull_request' && failure()
+        run: echo ::set-env name=IS_FORKED::$(if [ -z "$GITHUB_HEAD_REF" ]; then echo "no"; else echo "yes"; fi)
+        shell: bash
+
+      - name: Comment Pull Request
+        if: github.event_name == 'pull_request' && failure() && env.IS_FORKED == 'no'
```

Comment:
Don't we want IS_FORKED=yes for the comment action to trigger.

Reply:
If the repository is forked, then the action is going to fail with a permissions error, so we want it to only run if it's not forked.